cloud-reactor-api/src/main/java/com/sequenceiq/cloudbreak/cloud/event/platform/CreateCloudNetworkRequest.java

```diff
@@ -0,0 +1,63 @@
+package com.sequenceiq.cloudbreak.cloud.event.platform;
+
+import java.util.Set;
+
+import com.sequenceiq.cloudbreak.cloud.event.CloudPlatformRequest;
+import com.sequenceiq.cloudbreak.cloud.model.CloudCredential;
+import com.sequenceiq.cloudbreak.cloud.model.ExtendedCloudCredential;
+
+public class CreateCloudNetworkRequest extends CloudPlatformRequest<CreateCloudNetworkResult> {
```

Comment:
`CloudNetworkCreationRequest` would be grammatically a better name.

CreateCloudNetworkRequest is a verb which means: please, create me a cloud network request. However this is a request object containing data for cloud network creation.